src/styles/_catalog.scss

```diff
@@ -1,22 +1,33 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
+  display: grid;
+  justify-content: center;
   margin: 0 auto;
+  max-width: 944px;
```

Comment:
You don't need this, all cards will take only the required width, and with `justify-content: center` they will be centered